src/index.js

```diff
@@ -6,6 +6,8 @@ const { pushView } = require('./utils')
 
 module.exports = async function (req, res) {
   const { pathname, query } = url.parse(req.url, /* parseQueryString */ true)
+  res.setHeader('Access-Control-Allow-Origin', '*')
+  res.setHeader("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept")
```

Comment:
Yeah, I had not thought of that. I just copied this from another project, I tried it and it worked. I know at least the `Origin` header is necessary, not sure about `X-Requested-With`